docs/tokenizers.rst

```diff
@@ -0,0 +1,32 @@
+Tokenizers
```

Comment:
Can you add a link to `tokenizers.rst` from the main `index.rst` file?

Reply:
I'd also suggest building the docs locally on your machine to check that the hyperlinks/rendering all come out cleanly

Reply:
Yup, pycharm shows the individual docs, but I can check out how It looks on ReadtheDocs too

Reply:
done